terminus/geometry/latlon.py

```diff
@@ -38,15 +38,22 @@ def __eq__(self, other):
         if self.lat != other.lat:
             return False
         else:
-            return (self.lon == other.lon) or (abs(self.lon) == 180 and abs(other.lon) == 180) or abs(self.lat) == 90
+            return (self.lon == other.lon) or \
+                   (abs(self.lon) == 180 and abs(other.lon) == 180) or \
+                abs(self.lat) == 90
+
+    def __repr__(self):
+        return '(lat, lon) = (%s, %s)' % (self.lat, self.lon)
 
```

Comment:
A good thing to keep as a rule of thumb (when possible) is for the `__repr__` string to give you an expression that you can evaluate to re-generate the object. So e.g. if your implementation was

```
def __repr__(self):
    return 'LatLon(%s, %s)' % (self.lat, self.lon)
```

then `LatLon(1, 2)` would give you the string `"LatLon(1, 2)"`, which is something you can throw at a python console and get back a `LatLon(1, 2)` object.